src/JsRoot/components/JsRootGraph2D.tsx

```diff
@@ -23,8 +23,10 @@ export function JsRootGraph2D(props: { page: Page2D; title?: string }) {
 			histogram.fXaxis.fXmax = x[nxpoints - 1];
 			histogram.fXaxis.fTitle = `${page.axisDim1.name} [${page.axisDim1.unit}]`;
 
-			histogram.fYaxis.fXmin = y[0];
-			histogram.fYaxis.fXmax = y[nypoints - 1];
+			histogram.fMinimum = Math.min(...y);
+			// Apply a scaling factor to the maximum value of the Y-axis to ensure proper visualization
```

Comment:
thats a 2D histogram, it has X axis, Y axis and colorscale as Z. It doesn't makes sense to rescale its Z axis here - its color !

By reverting the change to this file you will the issue described here: https://github.com/yaptide/ui/pull/2066#issuecomment-2873097399